artichoke-backend/src/extn/core/integer/mod.rs

```diff
@@ -98,10 +98,13 @@ impl Integer {
 
     pub fn chr(self, interp: &mut Artichoke, encoding: Option<Value>) -> Result<Vec<u8>, Error> {
```

Comment:
This is the crux of the change I'd like us to make:

```suggestion
    pub fn chr(self, interp: &mut Artichoke, encoding: Option<Value>) -> Result<spinoso_string::String, Error> {
```